import-js.py

```diff
@@ -136,6 +136,8 @@ def handle_resolved_imports(resolved_imports):
                 args['command'] = 'add'
                 args['imports'] = resolved_imports
                 self.run(edit, **args)
+            self.view.run_command("import_js_replace",
+                                  {"characters": result.get('fileContent')})
             self.ask_to_resolve(result.get('unresolvedImports'),
                                 handle_resolved_imports)
             return
```

Comment:
That was my first thought, but that didn't work. I think we need to update the buffer before asking to resolve.